### DISCUSSION TITLE
Fall League Overview should show all of the Gold snakes

### DISCUSSION THREAD

**Discussion by BoldBigflank** [BattlesnakeOfficial/feedback#135] [General]
https://play.battlesnake.com/league/fall-league-2021/

The number is cut off at 50, but now that the gold league goes to 61 it would make sense for everyone in gold league to move "above the fold".

I realize this suggestion is subjective, so feel free to decide something else.

**Comment by bcambl** [comment]
The line you see at 61 marks the top 32 gold snakes for the tournament.

**Reply by BoldBigflank** [reply]
Sorry, I wasn't clear enough in my description. In the Overview page, the leaderboard is cut off at 50. It would make sense for the cutoff to be at the end of Gold in the Overview page.

**Reply by bcambl** [reply]
Ah, I understand what your saying. You are suggesting the overview to display the top 64 snakes (16+16+32) instead of top 50.

**Reply by bcambl** [reply]
..or perhaps showing all tournament qualified was my contribution to this idea. The overview is currently just  _top 50_ regardless of rank.

**Comment by bvanvugt** [comment]
Great call - we'll bump it to 75 for the time being. Should be updated this week.
